fidesctl/src/fidesctl/cli/__init__.py

```diff
@@ -81,6 +81,8 @@ def cli(ctx: click.Context, config_path: str, local: bool) -> None:
         config.cli.local_mode = True
 
     ctx.obj["CONFIG"] = config
+
+    # Run the help command if there isn't a subcommand
```

Comment:
I don't believe this comment adds value. The conditional feels pretty straightforward to me.